### PR TITLE
Setting documentation for InvertForPrinting

### PR DESCRIPTION
**Pull request by mcquin**

### CODE REVIEW

File: cellprofiler/modules/invertforprinting.py [hunk 1]
```diff
@@ -36,48 +36,189 @@ class InvertForPrinting(cpm.Module):
...
+            doc="Specify whether you are combining several grayscale images or loading a single color image."
+        )
+
+        self.wants_red_input = cps.Binary(
+            "Use a red image?",
+            True,
+            doc="""\
+*(Used only if input image type is "{CC_GRAYSCALE}")*
+
+Select *{YES}* to specify an image to use for the red channel.
```
Comment (bethac07) [inline cellprofiler/modules/invertforprinting.py]: `"*{YES}*"`

File: cellprofiler/modules/invertforprinting.py [hunk 2]
```diff
@@ -36,48 +36,189 @@ class InvertForPrinting(cpm.Module):
...
+            })
+        )
+
+        self.wants_green_input = cps.Binary(
+            "Use a green image?",
+            True,
+            doc="""\
+*(Used only if input image type is "{CC_GRAYSCALE}")*
+
+Select *{YES}* to specify an image to use for the green channel.
```
Comment (bethac07) [inline cellprofiler/modules/invertforprinting.py]: `"*{YES}*"`

File: cellprofiler/modules/invertforprinting.py [hunk 3]
```diff
@@ -36,48 +36,189 @@ class InvertForPrinting(cpm.Module):
...
+            })
+        )
+
+        self.wants_blue_input = cps.Binary(
+            "Use a blue image?",
+            True,
+            doc="""\
+*(Used only if input image type is "{CC_GRAYSCALE}")*
+
+Select *{YES}* to specify an image to use for the blue channel.
```
Comment (bethac07) [inline cellprofiler/modules/invertforprinting.py]: `"*{YES}*"`

File: cellprofiler/modules/invertforprinting.py [hunk 4]
```diff
@@ -36,48 +36,189 @@ class InvertForPrinting(cpm.Module):
...
-        self.blue_output_image = cps.ImageNameProvider("Name the blue image", "InvertedBlue")
-
-        self.color_output_image = cps.ImageNameProvider("Name the inverted color image","InvertedColor", doc="""\
+        self.output_color_choice = cps.Choice(
+            "Output image type",
+            CC_ALL,
+            doc="Specify whether you want to produce several grayscale images or one color image.")
+
+        self.wants_red_output = cps.Binary(
+            "Select *{YES}* to produce a red image.".format(**{"YES": YES}),
```
Comment (bethac07) [inline cellprofiler/modules/invertforprinting.py]: `"*{YES}*"`

File: cellprofiler/modules/invertforprinting.py [hunk 5]
```diff
@@ -36,48 +36,189 @@ class InvertForPrinting(cpm.Module):
...
+            True,
+            doc="""\
+*(Used only if output image type is "{CC_GRAYSCALE}")*
+
+Select *{YES}* to produce a grayscale image corresponding to the inverted red channel.
```
Comment (bethac07) [inline cellprofiler/modules/invertforprinting.py]: `"*{YES}*"`

File: cellprofiler/modules/invertforprinting.py [hunk 6]
```diff
@@ -36,48 +36,189 @@ class InvertForPrinting(cpm.Module):
...
+*(Used only if output image type is "{CC_GRAYSCALE}" and a red image is output)*
+
+Provide a name for the inverted red channel image.
+""".format(**{
+                "CC_GRAYSCALE": CC_GRAYSCALE
+            })
+        )
+
+        self.wants_green_output = cps.Binary(
+            "Select *{YES}* to produce a green image.".format(**{"YES": YES}),
```
Comment (bethac07) [inline cellprofiler/modules/invertforprinting.py]: `"*{YES}*"`

File: cellprofiler/modules/invertforprinting.py [hunk 7]
```diff
@@ -36,48 +36,189 @@ class InvertForPrinting(cpm.Module):
...
+            True,
+            doc="""\
+*(Used only if output image type is "{CC_GRAYSCALE}")*
+
+Select *{YES}* to produce a grayscale image corresponding to the inverted green channel.
```
Comment (bethac07) [inline cellprofiler/modules/invertforprinting.py]: `"*{YES}*"`

File: cellprofiler/modules/invertforprinting.py [hunk 8]
```diff
@@ -36,48 +36,189 @@ class InvertForPrinting(cpm.Module):
...
+*(Used only if output image type is "{CC_GRAYSCALE}" and a green image is output)*
+
+Provide a name for the inverted green channel image.
+""".format(**{
+                "CC_GRAYSCALE": CC_GRAYSCALE
+            })
+        )
+
+        self.wants_blue_output = cps.Binary(
+            "Select *{YES}* to produce a blue image.".format(**{"YES": YES}),
```
Comment (bethac07) [inline cellprofiler/modules/invertforprinting.py]: `"*{YES}*"`

File: cellprofiler/modules/invertforprinting.py [hunk 9]
```diff
@@ -36,48 +36,189 @@ class InvertForPrinting(cpm.Module):
...
+            True,
+            doc="""\
+*(Used only if output image type is "{CC_GRAYSCALE}")*
+
+Select *{YES}* to produce a grayscale image corresponding to the inverted blue channel.
```
Comment (bethac07) [inline cellprofiler/modules/invertforprinting.py]: `"*{YES}*"`

**Comment by mcquin** [comment]
@bethac07 concerns have been addressed... merging!

**Pull request merged**